crypto/x509v3/v3_ncons.c

```diff
@@ -9,6 +9,7 @@
 
 #include <stdio.h>
 #include "internal/cryptlib.h"
+#include "e_os.h"
```

Comment:
A new PR is good.  There are several instances where e_os.h is included solely for the string functions.  I added some comments to this effect but I'm pretty sure I didn't do all of them.

Breaking e_os.h into a manageable number of other internal include files makes sense to me.  Each inclusion would be more specific about what it requires and it solves the whole monolithic problem this began with.  The key is manageable, it would easily get out of hand.  My `nelem.h` is probably an example of this already :(